hip-XXXX.md

```diff
@@ -0,0 +1,160 @@
+# HIP-0000 : DANE for Decentralized Naming Systems (dDANE)
+
+```
+Number:  HIP-XXXX
+Title:   DANE for Decentralized Naming Systems (dDANE)
+Type:    Informational
+Status:  Draft
+Authors: Matthew Zipkin (@pinheadmz)
+         Hasan Adams (@buffrr)
+Created: 2021-06-17
+```
+
+## Abstract
+
+ICANN-rooted DNS is not the only internet naming system available to users, but it
+is the only naming system in which certificate authorities issue SSL certificates.
+Since SSL certificates are the mechanism behind HTTPS, browsing securely to websites
+outside the ICANN-rooted DNS is difficult and out of scope for the majority of users.
+
+DANE is a protocol that uses DNSSEC to authenticate SSL certificates as presented by
+a web server and could be used to bypass certificate authorities, assuming the naming
+system is trusted or cryptographically secure.
+
+DANE is an old innovation, initially proposed in 2012 in [RFC6698] and updated in 2015
+in [RFC7671]. It has been met with consistent rejection, particularly from web browsers:
+[1] [2] [3] [4] The arguments against DANE/DNSSEC adoption by web browsers can be summarized
+as follows:
+
+1. DNSSEC is unnecessary
+2. DNSSEC is a government-controlled PKI
+3. DNSSEC is cryptographically weak
+4. DANE is slow
+
+Note that these widely-shared critiques are themselves over six years old, and there
+is a new exciting technology that can breathe new life into the DANE use-case: BLOCKCHAIN.
+In particular, there are many blockchain-based domain naming systems in production today,
+including [Handshake], [Namecoin], [Ethereum Name Service], [Unstoppable Domains],
+[yat], and [Butterfly Protocol]. **Every single one of these domain name systems
+needs DANE support in web browsers or their users will have no security. Without
+convenient and secure web browsing accessible to the average user, none of these
+systems will ever gain wide adoption.**
+
+Decentralized, blockchain-based naming systems offer a brand new security model and
+a brand new class of users and services, meaning that there is no "legacy" infrastructure
+to update. If we establish a high-quality standard right now, there will be no
+resistance from old systems that refuse to upgrade. Therefore:
+
+1. DNSSEC is necessary for DANE, which means secure browsing to websites hosted on
+decentralized naming systems.
+2. DNSSEC is not government-controlled if the domain name root zone is secured by a
+decentralized consensus mechanism like [proof-of-work].
+3. DNSSEC can be strong if all participants agree to require high-quality cryptosystems
+4. DANE is slow: decentralized systems are never convenient and there are difficult
```

Comment:
i think it might be worth noting that there are probably ways to optimize around the slowness problem that would be acceptable, e.g. concurrency, prefetching, caching kinds of stuff. i seem to vaguely recall seeing an existing RFC about that too

Reply:
Ok thanks will add.